app/controllers/answers.js

```diff
@@ -26,15 +27,15 @@ exports.answer = function(req, res, next, id) {
 /**
  * Show an answer
  */
-exports.show = function(req, res) {
+exports.show = (req, res) => {
     res.jsonp(req.answer);
 };
 
 /**
  * List of Answers
  */
-exports.all = function(req, res) {
-    Answer.find({ official: true }).select('-_id').exec(function(err, answers) {
+exports.all = (req, res) => {
+    Answer.find({ official: true }).select('-_id').exec((err, answers) => {
         if (err) {
```

Comment:
For chains, put methods on separate lines for clearer readability.
.find
.select
.exec